src/components/Button.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components/macro";
+
+export default function Button({icon, text}){
+	return(
+		<StyledButton>
```

Comment:
Usually a button has an onClick which you want to pass as props. I would already implement it here.
```suggestion
export default function Button({icon, text, onClick}){
	return(
		<StyledButton onClick={onClick}>
```